scRNA-seq/setup/scRNA_data_setup.md

```diff
@@ -0,0 +1,51 @@
+# Single Cell RNA-seq training data setup"
```

Comment:
How do you expect this file to be viewed typically? Local copy in a participant's home directory or on GitHub? (This is, in essence, a question about naming this file `README.md`.)

Reply:
Hmmm... that is a good idea, I hadn't really considered it.